meta-balena-common/recipes-kernel/linux/kernel-devsrc.bb

```diff
@@ -150,6 +150,9 @@ do_install() {
             cp -a --parents arch/arm64/kernel/vdso/sigreturn.S $kerneldir/build/
             cp -a --parents arch/arm64/kernel/vdso/note.S $kerneldir/build/
             cp -a --parents arch/arm64/kernel/vdso/gen_vdso_offsets.sh $kerneldir/build/
+	    if [ -f "arch/arm64/kernel/vdso/vdso.lds.S" ]; then
```

Comment:
Please fix indentation.

Reply:
Hey @agherzan, my chunk was inserted between 2 lines with different indentation. My chunk is aligned with the indentation of the line below

Reply:
Gotcha.